assets/js/components/coverage/Constants.js

```diff
@@ -92,6 +92,7 @@ export const getColumns = (
                 if (
                   warnUnfollow &&
                   !!hotspot_claimed &&
```

Comment:
maybe this should check for hotspot_preferred instead of hotspot_claimed? Since this can only run if the hotspot is preferred right?